ouster-ros/src/os_sensor_node_base.cpp

```diff
@@ -74,4 +82,73 @@ bool OusterSensorNodeBase::write_text_to_file(const std::string& file_path,
     return true;
 }
 
+std::string OusterSensorNodeBase::transition_id_to_string(uint8_t transition_id) {
+    switch (transition_id) {
+        case lifecycle_msgs::msg::Transition::TRANSITION_CREATE:
+            return "create"s;
+        case lifecycle_msgs::msg::Transition::TRANSITION_CONFIGURE:
+            return "configure"s;
+        case lifecycle_msgs::msg::Transition::TRANSITION_CLEANUP:
+            return "cleanup"s;
+        case lifecycle_msgs::msg::Transition::TRANSITION_ACTIVATE:
+            return "activate";
+        case lifecycle_msgs::msg::Transition::TRANSITION_DEACTIVATE:
+            return "deactivate"s;
```

Comment:
Why the random usage of s?

Reply:
I think i am missing the `s` on "activate", this is syntactic sugar for string{""};